docs/tools/aiven-console.rst

```diff
@@ -0,0 +1,69 @@
+Aiven Console
```

Comment:
I believe we need to add `Aiven Console` in the file `.githube/vale/styles/capitalization_headings.yml`

Reply:
Thanks for the info, I think I trying to fix the error in the wrong file.